clients/cpp-console/README.md

```diff
@@ -32,3 +32,50 @@ Check out how to [compile this C++ console sample for Linux on a Windows machine
 ## Building for Windows
 
 Check out the [Windows.md](docs/Windows.md) for detailed instructions.
+
+## Quickstart
+
+1. Follow the instructions listed above to setup the building and running environment. Besides, get subscription key and key region ready at hand, along with app id if you are using a Custom Commands application.
+
+2. Build the executable from source code:
+* First clone the repository:
+```cmd
+git clone https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistants.git
+```
+* Then change directories if you are doing for Windows:
+```cmd
+cd Cognitive-Services-Voice-Assistants\clients\cpp-console\src\windows
+```
+* Launch Visual Studio 2017 or newer by opening the solution cppSample.sln. Build the solution (the default build flavor is Debug x64).
+
+3. Config and run the executable.
+* For Debug x64 build, go to the directory:
+```cmd
+cd Cognitive-Services-Voice-Assistants\clients\cpp-console\src\windows\x64\Debug
+```
+* Open config.json (if it is not there, copy and paste from Cognitive-Services-Voice-Assistant\clients\cpp-console\configs), replace YOUR_SUBSCRIPTION_KEY, YOUR_SUBSCRIPTION_REGION, and YOUR_CUSTOM_COMMANDS_APP_ID with actual values. The KeywordRecognitionModel, Keyword, Volume, TTSBargeInSupported, CustomMicConfigPath, LinuxCaptureDeviceName below show you how these values could be.
+```json
+{
+  "KeywordRecognitionModel": "/data/cppSample/computer.table",
+  "Keyword": "Computer",
+  "SpeechSubscriptionKey": "YOUR_SUBSCRIPTION_KEY",
+  "SpeechRegion": "YOUR_SUBSCRIPTION_REGION",
+  "Volume": "25",
+  "CustomCommandsAppId": " YOUR_CUSTOM_COMMANDS_APP_ID",
+  "CustomVoiceDeploymentIds": "",
+  "SpeechSDKLogFile": "",
+  "TTSBargeInSupported": "true",
+  "CustomMicConfigPath" :"/home/ubuntu/cpp-console/configs/micConfig.json",
+  "LinuxCaptureDeviceName": "hw:1,0"
+}
```

Comment:
Remove fields that are not relevant for Windows:
  "CustomMicConfigPath" :"/home/ubuntu/cpp-console/configs/micConfig.json",
  "LinuxCaptureDeviceName": "hw:1,0"

Also, this has LINUX folder path syntax with "/". I don't think this will work on Windows:
"KeywordRecognitionModel": "/data/cppSample/computer.table",

Reply:
Updated.